lambda-runtime/README.md

```diff
@@ -0,0 +1,14 @@
+# lambda-runtime
+
+An opinionated runtime environment for haskell applications running on [AWS Lambda].
+
+Key Features:
+
+- Only exposes a single function that constructs a lambda
+- Strong emphasis on error catching and handling, exposing the ability to catch `InternalLambdaClientErrors` to user applications for post processing and reporting.
+- Aim for small dependence footprint
+- Expose a runCLI function which can be used to test lambda locally see example executable
```

Comment:
@epicallan I'd put `runCLI` between backticks for canonical markdown rendering of code.

Reply:
__comment:__ I would probably add a period after "locally" and capitalize "See", eg:

```
- Expose a `runCLI` function which can be used to test lambda locally. See example executable.
```

Also ideally "example executable" would also be hyperlinked to the source code for it.

Reply:
I have made these edits and removed the `See example..` repetition. Should be in the next update.